internal/deploy/docker.go

```diff
@@ -226,14 +249,26 @@ func (c *dockerDeploymentManifest) Remove(profile ServiceRequest, services []Ser
 }
 
 // Start a container
-func (c *dockerDeploymentManifest) Start(service ServiceRequest) error {
+func (c *dockerDeploymentManifest) Start(ctx context.Context, service ServiceRequest) error {
+	span, _ := apm.StartSpanOptions(ctx, "Starting service from compose deployment", "docker-compose.service.start", apm.SpanOptions{
+		Parent: apm.SpanFromContext(ctx).TraceContext(),
+	})
+	span.Context.SetLabel("service", service)
+	defer span.End()
+
 	manifest, _ := c.Inspect(context.Background(), service)
-	_, err := shell.Execute(c.Context, ".", "docker", "start", manifest.Name)
```

Comment:
@adam-stokes do you think we still need to store a context in the dockerDeploymentManifest struct?